sdk/include/psdk/versionhelpers.h

```diff
@@ -148,9 +148,6 @@ IsActiveSessionCountLimited()
 VERSIONHELPERAPI
 IsReactOS()
 {
-    // FIXME: Find a better method!
-    WCHAR szWinDir[MAX_PATH];
-    GetWindowsDirectoryW(szWinDir, _countof(szWinDir));
-    return (wcsstr(szWinDir, L"ReactOS") != NULL);
+    return *(UINT*)(MM_SHARED_USER_DATA_VA + PAGE_SIZE - sizeof(ULONG)) == 0x8EAC705;
```

Comment:
You might need `mmtypes.h` for it.
If that does not work, copy the define for `MM_SHARED_USER_DATA_VA` locally and we can check out why that would be needed later.